service/src/main.rs

```diff
@@ -61,26 +62,24 @@ async fn main() -> Result<(), std::io::Error> {
     // Make an instance of network subgraph at either
     // graph_node_query_endpoint/subgraphs/id/network_subgraph_deployment
     // or network_subgraph_endpoint
-    let network_subgraph = NetworkSubgraph::new(
+    let network_subgraph: &'static NetworkSubgraph = Box::leak(Box::new(NetworkSubgraph::new(
```

Comment:
This works, but since NetworkSubgraph is ARC, it could also have been cloned around. Since it indeed is supposed to stick around the whole lifetime of the program it does not technically make a big difference though.

Reply:
Perhaps I should have used this pattern since the beginning though, makes a lot of things even simpler. I suppose the only downside is that it makes everything expect a static reference, which can be sometimes limiting in a library?